spark/dl/pom.xml

```diff
@@ -3,7 +3,7 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <parent>
-        <artifactId>bigdl-parent</artifactId>
+        <artifactId>spark</artifactId>
```

Comment:
there are bigdl and spark-version in the file spark, then I add a pom.xml for spark file, and named a module spark. So spark module is bigdl module and spark-version module's parent. Do we need add such spark module?